_inc/client/my-plan/checklist-progress-card/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ProgressBar from './progress-bar';
+import QueryChecklistProgress from 'components/data/query-checklist-progress';
+import QuerySite from 'components/data/query-site';
+import { getSiteID } from 'state/site';
+import { getTasks } from 'state/checklist/selectors';
+
+// Style imports are unused
+// import './style.scss';
```

Comment:
Yeah… I was dreaming of a day when the build system uses webpack-based sass… I'll remove them.

Reply:
These lines should be cleaned out?